src/api/users.ts

```diff
@@ -0,0 +1,38 @@
+import { Comment } from '../types/Comment';
+import { Post } from '../types/Post';
+import { User } from '../types/User';
+import { client } from '../utils/fetchClient';
+
+export const getUsers = () => {
+  return client.get<User[]>(`/users`);
+};
+
+export const getPostsByUser = (userId = 0) => {
+  return client.get<Post[]>(`/posts?userId=${userId}`);
```

Comment:
The default parameter `userId = 0` might lead to unintended API calls if not handled properly. Ensure that this default value is intentional and that the API can handle requests with `userId = 0` appropriately.